lib/tasks/db_enhancements.rake

```diff
@@ -18,7 +18,7 @@ db_namespace = namespace :db do
     ActiveRecord::Base.configurations.configs_for(env_name: Rails.env).each do |db_config|
       ActiveRecord::Base.establish_connection(db_config.configuration_hash)
       unless ActiveRecord::Base.connection.table_exists? 'ar_internal_metadata'
-        db_namespace['load_config'].invoke if ActiveRecord::Base.schema_format == :ruby
+        db_namespace['load_config'].invoke if Rails.application.config.active_record.schema_format == :ruby
```

Comment:
```suggestion
        db_namespace['load_config'].invoke if ActiveRecord.schema_format == :ruby
```

ref: https://github.com/thuss/standalone-migrations/issues/169 

`Rails.application.config.active_record.schema_format` will return nil ( tested on rails console) since we aren't setting the schema_format in our rails config.